pkg/kapp/cmd/app/delete.go

```diff
@@ -30,6 +30,11 @@ type DeleteOptions struct {
 	ResourceTypesFlags  ResourceTypesFlags
 }
 
+type changeFlags struct {
+	hasNoChanges     bool
```

Comment:
should be public since it's being accessed outside of the struct.

Reply:
Done